pycc/ccwfn.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, scf_wfn, **kwargs):
         # models requiring T1-transformed integrals
         self.need_t1_transform = ['CC3']
 
-        valid_local_models = [None, 'LPNO', 'PAO']
+        valid_local_models = [None, 'LPNO', 'PAO','LPNOpp']
```

Comment:
RE: previous group discussions, I believe we should change all of our "LPNO" methods to simply be "PNO", following F. Neese's nomenclature. this is a good time to make that change.